randovania/games/am2r/logic_database/Industrial Complex.txt

```diff
@@ -400,15 +400,21 @@ Extra - minimap_data: [{'x': 60, 'y': 22}, {'x': 60, 'y': 23}, {'x': 61, 'y': 22
   > Event - Torizo
       All of the following:
           Any of the following:
-              # Energy requirements
-              All of the following:
-                  # With space jump
-                  Space Jump
-                  Combat (Advanced) or Normal Damage ≥ 260
-              All of the following:
-                  # Without space jump
-                  Combat (Hypermode) or Normal Damage ≥ 399
-          # Fusion Mode requirements
+              # Weapon Requirements
+              Missiles ≥ 50 or Spazer Beam or Super Missiles ≥ 10 or Combat (Intermediate)
+              Power Bombs ≥ 5 and Can Use Power Bombs
+          Any of the following:
+              # Health Requirements
+              Combat (Advanced) or Normal Damage ≥ 300
+              Combat (Beginner) and Normal Damage ≥ 200
+              Combat (Intermediate) and Normal Damage ≥ 100
+          All of the following:
+              # Dodging Requirements
+              # Ground Phase
+              Hi-Jump Boots or Morph Ball or Space Jump or Walljump (Intermediate)
+              # Air Phase
+              Space Jump or Walljump (Advanced)
```

Comment:
i'm not sure i like this. this still means, that 300 health + WJ advanced is something that's in logic.
we probably need to do something where, 1) the above health reqs are only in effect with SJ, 2) we have a seperate amount of health reqs for NSJ, and 3) we lock NSJ behind a combat trick.

Reply:
I don't agree with split health for SJ/NSJ, this is much simpler to understand and maintain.

The difficulty with point 3) is that if you lock WJ behind combat, you are removing health from the requirements.
`WJ (adv) + Combat (adv)` NSJ would mean that you could be expected to do this damageless. But I think it's much more friendly if you say "I can do wall jumps but didnt turn combat on so I have some health"

Reply:
the seperate (N)SJ health requirements, was so that 3) doesnt conflict with the current requirements.
IMO it's very misleading if someone has walljumping set to expert, but is then expected to do NSJ torizo. because for me, NSJ torizo is a combat thing, not a walljump thing.